schemas/macro_service.schema.json

```diff
@@ -139,6 +157,10 @@
     "Response": {
       "$description": "A response to a [MacroRequest] or [HostRequest].",
       "properties": {
+        "requestId": {
+          "description": "The id of the [AugmentRequest] this is responding to.",
```

Comment:
I think that should be "id of the [MacroRequest] or [HostRequest]..."?

Reply:
I just changed it to the generic "request" since  that will not get out of date later